common-ui/src/main/res/values/design-system-typograhy.xml

```diff
@@ -94,7 +93,7 @@
 
     <style name="Typography.DuckDuckGo.Title">
         <item name="android:textSize">32sp</item>
-        <item name="android:lineSpacingExtra">4sp</item>
+        <item name="android:lineSpacingExtra">2sp</item>
```

Comment:
```suggestion
        <item name="android:lineSpacingExtra">4sp</item>
```
I think some typography changes were added by mistake after the merging `ads_release` into your branch